extensions/BugModal/web/new_bug.js

```diff
@@ -1,28 +1,28 @@
+var initial = {}
 var comp_desc = {}
 
-function load_products(query, callback) {
+$(document).ready(function() {
     bugzilla_ajax(
             {
-                url: 'rest/bug_modal/products'
+                url: 'rest/bug_modal/initial_loader'
             },
             function(data) {
-                callback(data.products);
+                initial = data
             },
             function() {
-                callback();
+                alert("Network issues. Please refresh the page and try again");
```

Comment:
How is this error message localized? Will any entered, unsubmitted, data be lost? Is it our responsibility to warn of that?